README.md

```diff
@@ -53,13 +53,13 @@ Here are the steps to have your company sign the CLA.
 1. Open a PR against a repo that uses EasyCLA.
 1. When blocked by the bot, follow the prompts and choose *Corporate*.
 1. Choose your company from the list. If it's not there, add it.
-1. Designate someone with signing authority (generally an officer or attorney, if in doubt ask your manager) to receive the DocuSign. This person is the *CLA Manager*.
+1. Designate someone with signing authority (generally an officer or attorney, if in doubt ask your manager) to receive the DocuSign form. This person is the *CLA Manager*.
```

Comment:
the change is correct; but the original is correct too - “the DocuSign” is colloquially used as the noun.